mesonbuild/build.py

```diff
@@ -572,6 +580,11 @@ def process_kwargs(self, kwargs, environment):
         if not isinstance(cslist, list):
             cslist = [cslist]
         self.add_compiler_args('cs', cslist)
+        cppstd = kwargs.get('cpp_std', None)
+        if not isinstance(cppstd, str) and cppstd != None:
+            raise InvalidArguments('cpp_std argument must be a string.')
+        if cppstd != None:
```

Comment:
__[flake8]__
* [E711] comparison to None should be 'if cond is not None:'

<!-- Comment by [SideCI](https://sideci.com) -->